clan_members_rank.py

```diff
@@ -2,9 +2,11 @@
 from multiprocessing import Pool
 
 from clash_royale_client import ClashRoyaleClient
-
+from common import schemas
 
 # NOTE: we want the highest level to come first (sort descending)
+
+

```

Comment:
/nit are the double blank here between comment and function necessary?

Reply:
this is autolint hmmmm

Reply:
double line before the comment so comment is right before the function